package.json

```diff
@@ -65,6 +65,7 @@
     "@pact-foundation/pact-node": "8.6.0",
     "@types/jasmine": "3.3.13",
     "@types/node": "12.0.8",
+    "ajv": "6.10.0",
```

Comment:
Should this be under `devDependencies`?

Reply:
No, because the missing peer warning occurs on consuming SPAs (so it will need to be installed automatically via `dependencies`).

Reply:
Got it, thanks 👍